Dockerfile

```diff
@@ -8,8 +8,8 @@
 #     http://www.apache.org/licenses/LICENSE-2.0
 #
 ARG DOCKER_REGISTRY
-ARG GO_VERSION=1.18
-ARG ALPINE_VERSION=3.16
+ARG GO_VERSION=1.20
```

Comment:
Could you please verify that 1.20 image has at least 1.20.3 golang version? There are quite a few CVE present in 1.20.0 that are fixed in 1.20.3.

Reply:
It is having 1.20.5 
```
docker run -it golang:1.20-alpine3.18 /bin/sh
/go # go version
go version go1.20.5 linux/amd64
/go # 
```